tests/test_r2s.py

```diff
@@ -308,6 +308,26 @@ def test_total_photon_source_intensity():
     intensity = total_photon_source_intensity(m, "source_density")
     assert_equal(intensity, 58)
 
+def test_total_photon_source_intensity_subvoxel():
```

Comment:
I think I understand this, but a comment block explaining the setup for this test would be very helpful if anyone needs to modify it in the future.